core/test/src/test_ppu.cpp

```diff
@@ -360,6 +362,131 @@ TEST_F(PpuTest, increment_vram_addr_by_32_after_reading) {
     EXPECT_EQ(expected, registers);
 }
 
+TEST_F(PpuTest, visible_two_sub_cycles) {
```

Comment:
The order would be clearer if this was tested with InSequence.

Reply:
I left the test for the whole line as is though.